### DISCUSSION TITLE
能否像openwrt中的mosdns一样，支持正则表达式配置hosts功能

### DISCUSSION THREAD

**Discussion by sk163** [kkkgo/PaoPaoDNS#170] [想法]
能否像openwrt中的mosdns一样，支持正则表达式配置hosts功能

比如regexp:^k8s-.*\.xxx\.zzz$ 10.10.10.1

**Comment by kkkgo** [comment]
*.xxx.zzz可以，目前支持的是以后缀匹配，这也符合域名的逐级匹配的模式。

如果有需求的话，后续的版本可以考虑增加支持。

**Reply by sk163** [reply]
非常感谢，很期待……因为有很多k8s-*.xxx.com的域名，目前只能一个一个配置，而且新增的域名也得一个一个配进来🙏

**Reply by kkkgo** [reply]
参考更新日志的Hosts模块。
https://github.com/kkkgo/PaoPaoDNS/discussions/187

**Reply by sk163** [reply]
RULES_TTL新增通配匹配测试了一下，非常好用，感谢感谢🙏！
